src/configMeta.ts

```diff
@@ -22,4 +33,42 @@ export default [
     hidden: true,
     encrypted: true,
   },
+  {
+    key: ConfigVars.LOCAL_DEV_SERVER_PORT,
+    description: 'The port number of the local dev server',
+    input: {
+      validator: (value: ConfigValue): boolean => {
+        if (!value) {
+          return false;
+        }
+
+        const parsedPort = parseInt(value as string, 10);
+
+        if (isNaN(parsedPort) || parsedPort < 1 || parsedPort > 65535) {
+          return false;
+        }
+        return true;
+      },
+      failedMessage: 'Must be a number between 1 and 65535',
```

Comment:
nit: consider putting this in a message file so we don't have to audit the entire repo later to take out hard-coded strings.

Reply:
done